substra/sdk/backends/local/backend.py

```diff
@@ -89,6 +97,49 @@ def _is_local(key: str):
         """
         return key.startswith("local_")
 
+    def __copy_file(self,
```

Comment:
I put __ because it was called by private functions, I changed it and moved the function to the DataAccess class, so it really manages the file saving, remote and local (https://github.com/SubstraFoundation/substra/pull/206#discussion_r463614693)

Reply:
Thanks!